library/ajax/pharmacy_autocomplete/search.php

```diff
@@ -0,0 +1,49 @@
+<?php
```

Comment:
Remove the above files:
library/ajax/pharmacy_autocomplete/js/jquery-1.9.1.min.js
library/ajax/pharmacy_autocomplete/js/jquery-ui.min.css
library/ajax/pharmacy_autocomplete/js/jquery-ui.min.js